src/endpoints/nfts/nft.controller.ts

```diff
@@ -121,6 +121,31 @@ export class NftController {
     return token;
   }
 
+  @Get('/nfts/:identifier/thumbnail')
+  @ApiResponse({
+    status: 200,
+    description: 'Non-fungible / semi-fungible token details',
+    type: Nft,
```

Comment:
this NFT type can be removed because this endpoint does not return a Nft type response